flexmeasures/data/models/planning/storage.py

```diff
@@ -588,19 +612,30 @@ def compute(self, skip_validation: bool = False) -> SchedulerOutputType:
         storage_schedule = fallback_charging_policy(
             sensor, device_constraints[0], start, end, resolution
         )
+        soc_schedule = integrate_time_series(storage_schedule, soc_at_start)
```

Comment:
Also needs conversion efficiencies and the storage efficiency to be passed.

Reply:
I have got derivative up, down efficiencies and the storage efficiency from `device_constraints`